database/index.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/test')
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => { console.log('connected to db!') });
+
+let imageSchema = new mongoose.Schema({
+  fullSizeURL: String,
+  thumbnailURL: String
```

Comment:
You may want to set up ESLint with Airbnb, since a trailing comma would be advised here after the last key pair value.

Reply:
Same goes for any object this file and probably other files.